README.md

```diff
@@ -1296,6 +1298,51 @@ echo "https://${GATEWAY_URL}/products"
 
 Make several requests to the URL for `/products` within a five second period to see requests fail with a status `429 Too Many Requests`.
 
+### Monitor your applications using Application Live View
+
+[Application Live View for VMware Tanzu](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.2/tap/GUID-app-live-view-about-app-live-view.html) is a lightweight insights and troubleshooting tool that helps app developers and app operators look inside running apps.
+
+Application Live View only supports Spring Boot applications.
+
+#### Enable Application Live View
```

Comment:
App Live View is already enabled by this point, let's rename this to "Access Application Live View"